### ISSUE TITLE
MapView occupies whole react-native-cardview space

### ISSUE THREAD

**Issue by san37puj** [fqborges/react-native-maps#10]
### Is this a bug report?

Yes

### Have you read the [Installation Instructions](https://github.com/react-community/react-native-maps/blob/master/docs/installation.md)?

Yes

### Steps to Reproduce

I was using react-native-maps to display google maps in my app. The layout hirarchy was as below:
  
   - CardView from 'react-native-cardview' ==> style={cardviewContainer}
       |- View from 'react-native' ==> style={defaultContainerStyle}
           |- View from 'react-native'  ==> header
           |- View from 'react-native' ==> style={mapContainer}
               |- MapView from 'react-native-maps' [provider='google'] ==> style={map}

Styles:
mapContainer: {
    marginLeft: -20,
    marginRight: -20,
    marginTop: 20,
    marginBottom: -20,
    height: 300,
  }
map: {
    ...StyleSheet.absoluteFillObject,
}
defaultContainerStyle: {
    padding: 20,
    paddingTop: 25,
    paddingBottom: 25,
}
cardviewContainer: {
    backgroundColor: '#fff',
    marginLeft: -2,
    marginRight: -2,
    marginBottom: 10,
  }

With above components and their cards map was rendering within required area. 
But, when I changed same above configuration with below, the OSM map accoupies whole CardView area. Here the problem is the Card header is not displayed/overlapped by map.

   - CardView from 'react-native-cardview' ==> style={cardviewContainer}
       |- View from 'react-native' ==> style={defaultContainerStyle}
           |- View from 'react-native'  ==> header
           |- View from 'react-native' ==> style={mapContainer}
               |- MapView from '**react-native-maps-osmdroid**' [provider=**PROVIDER_OSMDROID**] ==> style={map}

1. 2. 3.

### Expected Behavior 

![using google maps](https://user-images.githubusercontent.com/51318448/64694280-f325fc00-d4b6-11e9-8902-22ce7cb7ad20.png)

### Actual Behavior

"Some header" and Icon is overlapped here.
![using osmdroid map](https://user-images.githubusercontent.com/51318448/64694309-0042eb00-d4b7-11e9-8808-ce1379ba8bc4.png)

### Reproducible Demo

[SampleGoogleMapApp.zip](https://github.com/fqborges/react-native-maps-osmdroid/files/3601106/SampleGoogleMapApp.zip)
![sample_app_with_google_map](https://user-images.githubusercontent.com/51318448/64703087-633d7d80-d4c9-11e9-9946-9c3ea0f11510.png)

[SampleOSMDroidMapApp.zip](https://github.com/fqborges/react-native-maps-osmdroid/files/3601107/SampleOSMDroidMapApp.zip)
![sample_app_with_osmdroid_map](https://user-images.githubusercontent.com/51318448/64703102-6a648b80-d4c9-11e9-9660-8133ca19edce.png)

**Comment by fqborges** [comment]
Yes, I noticed that before on some containers views. I did not have the time to debug that yet, but I guess it is related to how react-native merge multiple empty "Views" in a single native view.

No time to fix this right now, though.

**Comment by arieldll** [comment]
Hi all,

Any solution for this bug?

Best Regards

**Comment by san37puj** [comment]
Yes, a workaroud! :)

I just used CardView (instead of View) inside CardView as below:

CardView from 'react-native-cardview' ==> style={cardviewContainer}
|- View from 'react-native' ==> style={defaultContainerStyle}
|- View from 'react-native' ==> header
|- **CardView** from 'react-native' ==> style={mapContainer}
|- MapView from 'react-native-maps-osmdroid' [provider=PROVIDER_OSMDROID] ==> style={map}

**Comment by arieldll** [comment]
Hi all,

Working for me too.
Thank you @san37puj 

Best Regards,

**Comment by SanyaMartinez** [comment]
wrapping in ScrollView also helped
